src/renderer/components/kubeconfig-dialog/kubeconfig-dialog.tsx

```diff
@@ -89,9 +89,8 @@ export class KubeConfigDialog extends React.Component<Props> {
   }
 
   copyToClipboard = () => {
-    if (this.config && copyToClipboard(this.configTextArea)) {
-      Notifications.ok("Config copied to clipboard");
-    }
+    clipboard.writeText(this.config);
```

Comment:
No return value? Can it throw?

Reply:
The documentation doesn't say it can. And the return value is `void`.